src/test/java/de/retest/web/it/TestHealerCssSelectorIT.java

```diff
@@ -0,0 +1,113 @@
+package de.retest.web.it;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+
+import java.net.MalformedURLException;
+import java.util.stream.Stream;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.openqa.selenium.By;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+
+import de.retest.web.AttributesUtil;
+import de.retest.web.selenium.AutocheckingRecheckDriver;
+
+class TestHealerCssSelectorIT {
+
+	private static WebDriver driver;

```

Comment:
According to our best practices, it's fine to have fields in tests package private, just like the methods.

Reply:
I removed the visibility qualifier in this and also in PredicateBuilderTest.